etc/firebase-admin.api.md

```diff
@@ -923,7 +925,7 @@ export namespace messaging {
         title?: string;
         vibrate?: number | number[];
     }
-    {};
+        {};

```

Comment:
This is interesting :D

Reply:
Yeah, that's because of https://github.com/microsoft/rushstack/issues/2808. Hopefully it will get fixed on their end.